src/Analyzer.Utilities/DiagnosticHelpers.cs

```diff
@@ -98,5 +99,117 @@ public static string GetMemberName(ISymbol symbol)
             // For other language constructs
             return symbol.Name;
         }
+               
+        public static bool IsInvisibleOutsideAssemblyAtSymbolLevel(ISymbol symbol)
+        {
+            return SymbolIsPrivateOrInternal(symbol)
+                || SymbolIsProtectedInSealed(symbol);
```

Comment:
I am surprised to find that this function is never used in Analyzers. I will remove it.